package.json

```diff
@@ -13,8 +13,8 @@
     "clean": "rimraf dist/",
     "copyfiles": "copyfiles -u 1 public/* dist/public",
     "postinstall": "husky install",
-    "lint": "eslint . --ext .ts",
-    "lint:fix": "eslint . --ext .ts --fix",
+    "lint": "eslint ./src --ext .ts",
+    "lint:fix": "eslint ./src --ext .ts --fix",
```

Comment:
Same comment as [here](https://github.com/UK-Export-Finance/mdm-api/pull/44/commits/fffeedb020a499a444130a9069acaa1f9e13db67#r1128343818) - do we not want to list the `test` files as well?

Reply:
this is updated now

Reply:
this is updated now